packages/amplify-category-storage/src/provider-utils/awscloudformation/service-walkthroughs/dynamoDb-walkthrough.ts

```diff
@@ -96,7 +96,14 @@ export async function updateWalkthrough(context: $TSContext) {
 
   const headlessMigrate = context.input.options?.yes || context.input.options?.forcePush || context.input.options?.headless;
   if (!cliInputsState.cliInputFileExists()) {
-    if (headlessMigrate || (await prompter.yesOrNo('File migration required to continue. Do you want to continue?', true))) {
+    const docsLink = 'https://docs.amplify.aws/cli/migration/overrides';
+    const migrateResourceMessage = [
+      `Do you want to migrate ${AmplifyCategories.STORAGE} resource "${resourceName}" to support overrides?`,
+      chalk.red(`Recommended to try in a non-production environment first. Run "amplify env add" to create or clone an environment.`),
+      `Learn more about this migration: ${docsLink}`,
+    ].join(EOL);
```

Comment:
would it be worthwhile to put this in a shared location for all of these functions to use?

Reply:
Yeah, the only reason I haven't done it yet is because I couldn't decide on where it should go. Do you think something like this makes sense in cli-core?

Reply:
@edwardfoyle